ETABS_oM/Elements/Spandrel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BH.oM.Base;
+using BH.oM.Structure.Elements;
+
+namespace BH.oM.Adapters.ETABS.Elements
+{
+    public class Spandrel : BHoMObject
```

Comment:
The SetSpandrel API call in etabs has a boolean option to define a spandrel as MultiStory or not. If this is something we want to control, we might want to add that in here as well.

Reply:
i'm not familiar with that feature enough to make that call if it should go in or not, especially considering i think that now since we have made applying pier and spandrel labels trivial, it changes the paradigm quite a bit on how the results should be used/post processed so my guess is that 'mulitstory' feature would be a thing of the past.